frontend/src/scenes/error-tracking/groups/OverviewTab.tsx

```diff
@@ -38,7 +36,13 @@ export const OverviewTab = (): JSX.Element => {
                         event ? (
                             <div className="h-full overflow-auto">
                                 <div className="bg-bg-light p-1 flex justify-end border-b min-h-[42px]">
-                                    <ViewSessionButton event={event} />
+                                    {mightHaveRecording(event.properties) && (
+                                        <ViewRecordingButton
```

Comment:
maybe nicer to do this `mightHaveRecording` check inside the component and set `disabledReason` on the button

Reply:
I didn't want to pass the `properties` into the component because not ever instance has access to them. Have changed it around so `disabledReason` is passed in as a prop in the two cases we know the event properties